src/config/mod.rs

```diff
@@ -1452,7 +1452,6 @@ impl RaftDbConfig {
         opts.set_max_manifest_file_size(self.max_manifest_file_size.0);
         opts.create_if_missing(self.create_if_missing);
         opts.set_max_open_files(self.max_open_files);
-        opts.enable_statistics(self.enable_statistics);
```

Comment:
There are some place don't use `engine_factory` at all, and those cases will miss statistics.